src/components/newEvent/Overview.tsx

```diff
@@ -0,0 +1,247 @@
+        <strong className="block">Type</strong>
+        <div className="grid grid-cols-2 gap-4">
+          {options.map((option) => (
+            <button
+              key={option.value}
+              type="button"
+              onClick={() => handleSelect(option.value)}
+              className={`
+                flex flex-col items-start p-4 rounded-lg border-2 transition-all duration-200 ease-in-out
+                ${
+                  selectedType === option.value
+                    ? 'border-blue-500 bg-blue-50 text-blue-700'
+                    : 'border-gray-300 hover:border-gray-400'
+                }
+              `}
+            >
+              <div className="font-semibold text-lg">{option.label}</div>
+              <div className="text-left text-sm text-gray-500">
+                {option.value === 'online' ? ONLINE_TEXT : IN_PERSON_TEXT}
+              </div>
+            </button>
+          ))}
+        </div>
+      </div>
+    );
+  };
+
+export function Overview() {
+
+    const {isContinueEnabled, setIsContinueEnabled} = useContext(newEventContext)
+
+    const {
+        register,
+        handleSubmit,
+        formState: { errors },
+        setError,
+        watch,
+      } = useForm<FormData>({
+        resolver: zodResolver(UserSchema), // Apply the zodResolver
+      });
+
+    function submitForm() {
+        console.log('Sucesso!')
+    }
+
+    const handleTypeChange = (value:React.FormEvent<HTMLInputElement>) => {
+        console.log('Selected type:', value);
+        // Update your form state here
+      };
+
+    const eventName = watch("eventName")
+    const startsAt = watch("startsAt")
+    const startsAtTime = watch("startsAtTime")
+    const endsAt = watch("endsAt")
+    const endsAtTime = watch("endsAtTime")
+    const imageUrl = watch("imageUrl")
+
+    if (eventName && startsAt && startsAtTime && endsAt && endsAtTime && imageUrl && (!isContinueEnabled)) {
+        setIsContinueEnabled(true)
+    }
+    if ((!(eventName && startsAt && startsAtTime && endsAt && endsAtTime && imageUrl)) && isContinueEnabled) {
+        setIsContinueEnabled(true)
+    }
+
+
+
+    return (
+        <>
+        <form
+        className="block w-full"
+        onSubmit={handleSubmit(submitForm)}
+        >
+            <h2 className="text-4xl text-bold mb-16">
+                + Create New Event
+            </h2>
+            <div className="flex items-center justify-between">
+                <FormField
+                className="w-500px h-12 mt-1 pl-2 border-2 rounded-lg"
+                label="Event Name"
+                type="text"
+                placeholder="Text"
+                name="eventName"
+                register={register}
+                error={errors.eventName}
+                />
+                <div className="flex items-center justify-between w-56 ">
+                    <FormField
+                    className="w-32 h-10 mt-1 pl-2 bg-slate-100 rounded-lg"
```

Comment:
all FormFields here are using the same styling. I would create a custom component and create an array with the custom information

Reply:
Almost all fields have some differences, I think we can just keep it simple for now